editor.html

```diff
@@ -260,6 +260,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
 	    <script src="base64-js/base64.js"></script>
         <script src="hexlify.js"
             type="application/javascript"></script>
+        <script src="static/js/urlparser.js" type="application/javascript"></script>
```

Comment:
Minor nitpick, other PRs have started removing some of the `type="application/javascript"` attributes (well, the scripts have been replaced and the type not included). So we can do the same here: `<script src="urlparser.js"></script>`

Reply:
Done